packages/snowpack-plugin/src/index.ts

```diff
@@ -0,0 +1,65 @@
+import path from "path"
+import { getConfig } from "@lingui/conf"
+import { createCompiledCatalog, getCatalogs, getCatalogForFile } from "@lingui/cli/api"
+
+type LinguiConfigOpts = {
+  cwd?: string;
+  configPath?: string;
+  skipValidation?: boolean;
+}
+type SnowpackLoadOpts = {
+  filePath: string
+}
+function extractLinguiMessages(_?, linguiConfig: LinguiConfigOpts = {}) {
```

Comment:
```suggestion
function extractLinguiMessages(snowpackConfig?, linguiConfig: LinguiConfigOpts = {}) {
```
this would be more readable unless there are some linter warnings for unused variables